DSharpPlus/Entities/Channel/Message/DiscordMessageFile.cs

```diff
@@ -39,6 +39,15 @@ internal DiscordMessageFile(string fileName, Stream stream, long? resetPositionT
             this.ResetPositionTo = resetPositionTo;
         }
 
+        public DiscordMessageFile(string fileName, Stream stream, bool resetPosition = false)
+        {
+            this.FileName = fileName;
+            this.Stream = stream;
+            this.FileType = null;
+            this.ContentType = null;
+            this.ResetPositionTo = resetPosition ? 0 : null;
```

Comment:
do we really want to support this? shouldn't we just store the current position of the stream instead?

Reply:
I added this as that's how other parts of the code does it.

Reply:
cc @VelvetToroyashi we might want to consider removing this entirely